.github/workflows/update-packages-from-list.yml

```diff
@@ -18,15 +18,11 @@ jobs:
         with:
           repository: gentoo/gentoo
           path: gentoo
-          # Gentoo is quite a large repo, so limit ourselves to last
-          # quarter milion of commits. It is about two years worth of changes.
-          fetch-depth: 250000
```

Comment:
Please keep it - it's needed by the sync script to find out the hash of the last commit that made the changes to the package in Gentoo. I think that the number can be lowered at some point, but certainly not to just 1.

Reply:
Brought back fetch-depth